GitUI/CommandsDialogs/BrowseDialog/GitStatusMonitor.cs

```diff
@@ -255,8 +260,6 @@ void GitUICommands_PostCheckout(object sender, GitUIPostActionEventArgs e)
         private void StartWatchingChanges(string workTreePath, string gitDirPath)
         {
             // reset status info, it was outdated
-            GitWorkingDirectoryStatusChanged?.Invoke(this, new GitWorkingDirectoryStatusEventArgs());
```

Comment:
I think this comment applied to the removed code, and should be removed too.

Reply:
You're right. (First, I thought this comment applied to the following code, too.)